tests/cases/compiler/checkJsFiles_skipDiagnostics.ts

```diff
@@ -0,0 +1,33 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+
+// @fileName: a.js
+var x = 0;
+
+
+/// @ts-suppress
+x();

```

Comment:
I actually wanted this functionality in .ts a few days ago -- embedding a chunk of stable JS code in TS code.

Sometimes you need to embed code rather than have it as an external module, for encapsulation. Such as, I was writing a plugin that needed to use compression, and I've embedded LZW algo in my ts plugin.

Of course LZW algo in JS needed a bunch of `: any` kicks to pass TSC checks. Would be great to be able to exclude chunks of code in TS from checks in such cases.